db/migrate/20170531094819_move_conferences_to_organizations.rb

```diff
@@ -0,0 +1,21 @@
+class MoveConferencesToOrganizations < ActiveRecord::Migration
+  class TempConference < ActiveRecord::Base
+    self.table_name = 'conferences'
+  end
+
+  class TempOrganization < ActiveRecord::Base
+    self.table_name = 'organizations'
+  end
+
+  def change
+    add_reference :conferences, :organization, index: true
+    add_foreign_key :conferences, :organizations, dependent: :delete
```

Comment:
I am not sure how to check that.
Shouldn't this work the same:
`add_reference :conferences, :organization, foreign_key: true, index: true`

Looks like passing the `foreign_key: true` option to `add_reference` execute `add_foreign_key` statement as well:
https://github.com/rails/rails/blob/d09a9bc07ddaf3c570aced434ab64cf06860678c/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb#L668